nntrainer/graph/network_graph.cpp

```diff
@@ -488,19 +489,35 @@ int NetworkGraph::realizeGraph() {
       NN_RETURN_STATUS();
     }
 
-    if (l.getType() != OutputLayer::type) {
-      status = realizeMultiOutputType(l);
-      NN_RETURN_STATUS();
-    }
-
     // Flatten in TimeDistLayer is not supported.
     if (l.getFlatten() && l.getType() != TimeDistLayer::type) {
       status = realizeFlattenType(l);
       NN_RETURN_STATUS();
     }
   }
 
-  setOutputLayers();
+  try {
+    setOutputLayers();
```

Comment:
can this be simply moved to line 455, and then call here once again?

Reply:
You mean before realizing activation and others? In that case, we have to take care about output layers as well.